### PR TITLE
.Net: Gemini/restructure

### PR DESCRIPTION
**Pull request by Krzysztof318**
@RogerBarreto

### CODE REVIEW

File: dotnet/src/Connectors/Connectors.GoogleVertexAI/Core/Gemini/GeminiContent.cs
Comment (RogerBarreto) [inline dotnet/src/Connectors/Connectors.GoogleVertexAI/Core/Gemini/GeminiContent.cs:1]: Is there a reason why those buts are public and not internal only?

File: dotnet/SK-dotnet.sln [hunk 1]
```diff
@@ -610,7 +610,7 @@ Global
 		{BCDD5B96-CCC3-46B9-8217-89CD5885F6A2} = {0247C2C9-86C3-45BA-8873-28B0948EDC0C}
 		{1F96837A-61EC-4C8F-904A-07BEBD05FDEE} = {1B4CBDE0-10C2-4E7D-9CD0-FE7586C96ED1}
 		{6578D31B-2CF3-4FF4-A845-7A0412FEB42E} = {1B4CBDE0-10C2-4E7D-9CD0-FE7586C96ED1}
-		{648CF4FE-4AFC-4EB0-87DB-9C2FE935CA24} = {0247C2C9-86C3-45BA-8873-28B0948EDC0C}
```
Comment (RogerBarreto) [inline dotnet/SK-dotnet.sln:613]: You sure this change is necessary?

**Review by RogerBarreto — APPROVED** [review]

File: dotnet/SK-dotnet.sln (hunk 1 above)
Comment (Krzysztof318) [reply dotnet/SK-dotnet.sln:613]: I changed this because you place huggingface.unittest in same location.

Comment (Krzysztof318) [reply dotnet/SK-dotnet.sln:613]: It move googlevertex.unitest project from ai to ai/connectors

**Pull request merged**